raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -1266,13 +1268,21 @@ contract TokenNetwork is Utils {
         internal
         returns (address signature_address)
     {
+        uint256 message_type_id = 1;
+
+        // Length of the actual message: 20 + 32 + 32 + 32 + 32 + 32 + 32
+        string memory message_length = '212';
```

Comment:
Can't these here and below be string constants in the contract?

Reply:
I wanted to keep `message_length` close to the actual messages, for readability.
But I don't have a hard preference. Do you want me to set the lengths as string constants in the contract?